mw-webapp/src/service/auth/logIn.ts

```diff
@@ -1,9 +1,20 @@
-import {signInWithRedirect} from "firebase/auth";
+import {signInWithPopup} from "firebase/auth";
 import {auth, provider} from "src/firebase";
 
 /**
  * Call method for log in
  */
 export const logIn = async () => {
-  await signInWithRedirect(auth, provider);
+
+  /**
+   * TODO: IMPORTANT!
+   * Using signInWithPopUp calls Error: Cross-Origin-Opener-Policy policy would block the window.closed call.
+   * According
+   * https://stackoverflow.com/questions/76446840/cross-origin-opener-policy-policy-would-block-the-window-closed-call-error-while
+   * The problem is with Cross-Origin-Embedder-Policy header, which is by default should be set to unsafe-none.
+   * Also Firebase recommends the next steps to resolve this issue:
+   * https://firebase.google.com/docs/auth/web/redirect-best-practices
+   *
+   */
+  await signInWithPopup(auth, provider);
```

Comment:
so by comment it is not clear should we change something or not .  And if yes - than why. Imagine you someone will read this comment first time, or you will read it in 1 year

Reply:
Added comment/ Is it anough now?